src/menu.tsx

```diff
@@ -1,15 +1,15 @@
 import * as React from 'react';
 import classnames from 'classnames';
-import {assignInlineVars} from '@vanilla-extract/dynamic';
 import {ESC, TAB} from './utils/key-codes';
 import {cancelEvent, getPrefixedDataAttributes} from './utils/dom';
 import Overlay from './overlay';
 import * as styles from './menu.css';
-import {sprinkles} from './sprinkles.css';
+import {Portal, useWindowSize} from '.';
```

Comment:
inside mistica I think that we always import things from the component's file `./Portal` and not from the index

Reply:
yes please, this avoids circular dependencies